syntex_syntax/src/attr.rs

```diff
@@ -334,11 +334,11 @@ pub enum InlineAttr {
 pub fn find_inline_attr(diagnostic: Option<&Handler>, attrs: &[Attribute]) -> InlineAttr {
     attrs.iter().fold(InlineAttr::None, |ia,attr| {
         match attr.node.value.node {
-            MetaItemKind::Word(ref n) if *n == "inline" => {
+            MetaItemKind::Word(ref n) if n == "inline" => {
```

Comment:
just out of curiosity, did this actually break or was it just possible to omit the explicit deref now? Because I changed something in rustc related to comparison of `InternedString` with `&str`